micah-akpan/questioner-api
# #163229253 Change status code to appropriate server error HTTP status codes

#### What does this PR do?
Changes Status code to appropriate server error codes
#### Description of Task to be completed?
Change all 400 errors for server to status code 500
#### How should this be manually tested?
- clone repo
- `npm install`
- `npm run dev:start`
- Launch Postman, try out the API token (https://localhost:9999/api/v1/signup) to signup with the wrong credentials (say a missing password)
- You should get a 500 server error
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#163229253
#### Screenshots (if appropriate)
#### Questions:
